web/index.html

```diff
@@ -70,287 +69,350 @@ <h3 class="sub-header">
+                                                                @click="navigateTo(recipe.B);"
+                                                                @mouseover="loadChunk(getChunk(recipe.B))">
+                                                                {{ data.index[recipe.B][0] }}&nbsp;{{
+                                                                data.index[recipe.B][1] }}
+                                                            </td>
+                                                            <td>=</td>
+                                                            <td class="item"
+                                                                :class="{'item-used': !saveData.has(recipe.C)}"
+                                                                @click="navigateTo(recipe.C);"
+                                                                @mouseover="loadChunk(getChunk(recipe.C))">
+                                                                {{ data.index[recipe.C][0] }}&nbsp;{{
+                                                                data.index[recipe.C][1] }}
+                                                            </td>
+                                                        </tr>
+                                                    </table>
+                                                    <div v-if="currentHiddenToLength > 0">
+                                                        <div v-if="!showHiddenRecipes" class="text-link"
+                                                            @click="showHiddenRecipes = true">
+                                                            {{currentHiddenToLength}} entries hidden
+                                                        </div>
+                                                        <div v-else class="text-link"
+                                                            @click="showHiddenRecipes = false">
+                                                            {{currentHiddenToLength}} entries shown
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                            </transition>
+                                            <div v-else style="font-weight: bold; padding-top: 5px;">
+                                                No recipes found
+                                            </div>
                                         </div>
-                                    </div>
+                                    </transition>
                                     <div id="element-scroll-end"></div>
                                 </div>
                             </div>
                         </div>
                     </div>
-                    <transition name="slide-up" mode="out-in">
```

Comment:
Moved the nav bar away from here because it was glitching strangely during transitions since it is rendered with position:fixed, which is not that compatible with the vue transition